yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -413,19 +407,11 @@ export class Archiver extends EventEmitter implements ArchiveSource, Traceable {
         localBlockForDestinationProvenBlockNumber &&
         provenArchive === localBlockForDestinationProvenBlockNumber.archive.root.toString()
       ) {
-        const [localProvenEpochNumber, localProvenBlockNumber] = await Promise.all([
-          this.store.getProvenL2EpochNumber(),
-          this.store.getProvenL2BlockNumber(),
-        ]);
-        if (
-          localProvenEpochNumber !== Number(provenEpochNumber) ||
-          localProvenBlockNumber !== Number(provenBlockNumber)
-        ) {
+        const localProvenBlockNumber = await this.store.getProvenL2BlockNumber();
```

Comment:
Seems like this is only used this once, so don't seem necessary to put it in a const 🤷 Fine to keep though.